test/test.js

```diff
@@ -48,6 +48,20 @@ const validMsg = {
     "PkZ34BRVSmGG51vMXo4GvaoS/2NBc0lzdFoVv4wkI8E8zXv4QYyE5o2mPACKOcrhrLJpymLzqpoE70q78INuBg==.sig.ed25519",
 };
 
+const hmacMsg = {
+  previous: null,
+  sequence: 1,
+  author: "@EnPSnV1HZdyE7pcKxqukyhmnwE9076RtAlYclaUMX5g=.ed25519",
+  timestamp: 1624360181359,
+  hash: "sha256",
+  content: { type: "example" },
+  signature:
+    "w670wqnD1A5blFaYxDiIhPOTwz8I7syVx30jac1feQK/OywHFfrcLVw2S1KmxK9GzWxvKxLMle/jKjf2+pHtAg==.sig.ed25519",
+};
+
+const hmacKey = null;
+const hmacVal = "CbwuwYXmZgN7ZSuycCXoKGOTU1dGwBex+paeA2kr37U=";
```

Comment:
Both of these consts have the same purpose, could we call them `hmacKey1`, `hmacKey2` or something like that?

Reply:
Sure thing.